MaxText/utils/ckpt_conversion/README.md

```diff
@@ -0,0 +1,173 @@
+# Checkpoint conversion utilities
+
+This guide provides instructions for using the scripts that convert model checkpoints bidirectionally between Hugging Face and MaxText formats.
+
+## Supported models
+
+The following models are supported:
+
+- Gemma2 (2B, 9B, 27B).
+- Gemma3 multimodal (4B, 12B, 27B).
+- Qwen3 (0.6B, 4B, 8B, 14B, 32B).
+
+## Prerequisites
+- Hugging Face requires Pytorch.
+- Hugging Face model checkpoints requires local disk space.
+  - The model files are always downloaded to a disk cache first before being loaded into memory, [see](https://huggingface.co/docs/accelerate/en/concept_guides/big_model_inference). The default Hugging Face storage path is $HOME/.cache/huggingface/hub
+
+## Hugging Face to MaxText
+
+Use the `to_maxtext.py` script to convert a Hugging Face model into a MaxText checkpoint. The script will automatically download the specified model from the Hugging Face Hub.
+
+\*\**For a complete example, see the test script at [`end_to_end/tpu/qwen3/4b/test_qwen3.sh`](../../../end_to_end/tpu/qwen3/4b/test_qwen3.sh) and [`end_to_end/tpu/gemma3/4b/test_gemma3_unified.sh`](../../../end_to_end/tpu/gemma3/4b/test_gemma3_unified.sh).*
+
+### Usage
+
+The following command demonstrates how to run the conversion. You must provide your Hugging Face token as an environment variable or in the `MaxText/configs/base.yml` file.
+
+```bash
+python -m MaxText.utils.ckpt_conversion.to_maxtext MaxText/configs/base.yml \
+    model_name=<model-name> \
+    base_output_directory=<gcs-path-to-save-checkpoint> \
+    hf_access_token=<your-hf-token> \
+    use_multimodal=false \
+    scan_layers=false
+```
+
+**Key arguments:**
+
+  * `model_name`: The model identifier, which should be defined in `MaxText/utils/utils.py`.
+  * `scan_layers`: Indicates if the output checkpoint is scanned(scan_layers=true) or unscanned(scan_layers=false).
```

Comment:
please add this link under scanned https://github.com/AI-Hypercomputer/maxtext/blob/main/getting_started/checkpoints.md 

also, space before (

Reply:
same for to_huggingface below